.idea/misc.xml

```diff
@@ -5,11 +5,19 @@
       <map>
```

Comment:
`.idea` files aren't needed. Use gitignore to exclude such files and remove already committed files.